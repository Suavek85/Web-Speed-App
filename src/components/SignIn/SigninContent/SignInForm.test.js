import React from 'react'
import { shallow } from 'enzyme'
import SigninForm from './SignInForm'

describe('SigninForm snapshot', () => {

  it('should match the snapshot', () => {
    const wrapper = shallow(<SigninForm />)
    expect(wrapper).toMatchSnapshot()
  })

})