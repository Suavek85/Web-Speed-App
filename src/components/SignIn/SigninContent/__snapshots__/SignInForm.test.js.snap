// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SigninForm snapshot should match the snapshot 1`] = `
<div>
  <h2>
    I already have an account
  </h2>
  <span>
    Sign in with your email and password
  </span>
  <form
    onSubmit={[Function]}
  >
    <Input
      handleChange={[Function]}
      label="Email"
      name="email"
      required={true}
      type="email"
      value=""
    />
    <Input
      handleChange={[Function]}
      label="Password"
      name="password"
      required={true}
      type="password"
      value=""
    />
    <Button
      btnStyle="regular"
      isFormContent={true}
    />
  </form>
</div>
`;
